api/models/people.py

```diff
@@ -108,6 +105,10 @@ def capitalize_each_word(str_: str) -> str:
             update_fields=update_fields,
         )
 
+    @staticmethod
+    def capitalize_each_word(str_: str) -> str:
```

Comment:
Are you sure it should be part of PersonalInfo class? I advise to do some utils/helpers module and put it there, since it could be used somewhere else and it hasn't direct relation to PersonalInfo class

Reply:
Done